cl/search/views.py

```diff
@@ -514,41 +519,59 @@ def show_results(request: HttpRequest) -> HttpResponse:
                 if not is_bot(request):
                     async_to_sync(tally_stat)("search.results")
 
+                    # Perform the search
+                    search_type = request.GET.get("type", SEARCH_TYPES.OPINION)
```

Comment:
Is there a reason you tweaked the case statement into an if/else, and moved it up here?